packages/mobile/src/app/saga.test.ts

```diff
@@ -1,29 +1,70 @@
+import i18n from 'i18next'
+import DeviceInfo from 'react-native-device-info'
 import { expectSaga } from 'redux-saga-test-plan'
 import * as matchers from 'redux-saga-test-plan/matchers'
-import { select } from 'redux-saga/effects'
+import { EffectProviders, StaticProvider } from 'redux-saga-test-plan/providers'
+import { call, select } from 'redux-saga/effects'
 import { WalletConnectPairingOrigin } from 'src/analytics/types'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
-import { appLock, openDeepLink, openUrl, setAppState } from 'src/app/actions'
-import { handleDeepLink, handleOpenUrl, handleSetAppState } from 'src/app/saga'
-import { getAppLocked, getLastTimeBackgrounded, getRequirePinOnAppOpen } from 'src/app/selectors'
+import {
+  appLock,
+  openDeepLink,
+  openUrl,
+  setAppState,
+  setOtaTranslationsLastUpdate,
+} from 'src/app/actions'
+import { currentLanguageSelector } from 'src/app/reducers'
+import {
+  handleDeepLink,
+  handleFetchOtaTranslations,
+  handleOpenUrl,
+  handleSetAppState,
+} from 'src/app/saga'
+import {
+  allowOtaTranslationsSelector,
+  getAppLocked,
+  getLastTimeBackgrounded,
+  getRequirePinOnAppOpen,
+  otaTranslationsAppVersionSelector,
+  otaTranslationsLanguageSelector,
+  otaTranslationsLastUpdateSelector,
+} from 'src/app/selectors'
 import { handleDappkitDeepLink } from 'src/dappkit/dappkit'
 import { receiveAttestationMessage } from 'src/identity/actions'
 import { CodeInputType } from 'src/identity/verification'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { handlePaymentDeeplink } from 'src/send/utils'
 import { navigateToURI } from 'src/utils/linking'
+import { saveOtaTranslations } from 'src/utils/otaTranslations'
 import { initialiseWalletConnect } from 'src/walletConnect/saga'
 import { selectHasPendingState } from 'src/walletConnect/selectors'
 import { handleWalletConnectDeepLink } from 'src/walletConnect/walletConnect'
 
+jest.mock('@crowdin/ota-client', () => {
+  return function () {
+    return {
+      getManifestTimestamp: jest.fn(() => 123456),
+      getLanguageMappings: jest.fn(),
+      getStringsByLocale: jest.fn(() => ({
+        someLang: { someNamespace: { someKey: 'someValue ' } },
+      })),
+    }
+  }
+})
+
+jest.mock('i18next', () => ({
+  addResourceBundle: jest.fn(),
+}))
+
 jest.mock('src/utils/time', () => ({
   clockInSync: () => true,
 }))
 
 jest.mock('src/dappkit/dappkit')
 
-const MockedAnalytics = ValoraAnalytics as any
+const MockedAnalytics = ValoraAnalytics as jest.Mocked<typeof ValoraAnalytics>
+const MockedI18n = i18n as jest.Mocked<typeof i18n>
```

Comment:
nit: There's an even more simple syntax:
```suggestion
import { mocked } from 'ts-jest/utils'

const mockedAnalytics = mocked(ValoraAnalytics)
const mockedI18n = mocked(i18n)
```

Reply:
oh cool i didn't know this existed! super nice, i have updated :)